src/main/java/io/jenkins/plugins/orka/OrkaAgent.java

```diff
@@ -143,6 +150,67 @@ public String getJvmOptions() {
         return this.jvmOptions;
     }
 
+    public List<PortMapping> getPortMapping() {
+        return portMapping;
+    }
+
+    // Method to process the port mappings
+    public void processPortMappings() {
+        for (PortMapping mapping : portMapping) {
+            int fromPort = mapping.getFrom();
+            int toPort = mapping.getTo();
+
+            // Example logic for processing
+            if (isValidPortRange(fromPort, toPort)) {
+                System.out.println("Mapping from port " + fromPort + " to port " + toPort);
+            } else {
+                System.out.println("Invalid port mapping: from " + fromPort + " to " + toPort);
+            }
+        }
+    }
+
+    private boolean isValidPortRange(int from, int to) {
+        return from >= 1024 && from <= 65535 && to >= 1024 && to <= 65535;
+    }
+
+    public String getPortMappingAsString() {
+        if (portMapping == null || portMapping.isEmpty()) {
+            return "";
+        }
+    
+        StringBuilder sb = new StringBuilder();
+        for (PortMapping mapping : portMapping) {
+            if (sb.length() > 0) {
+                sb.append(",");
+            }
+            sb.append(mapping.getFrom()).append(":").append(mapping.getTo());
+        }
+        return sb.toString();
+    }
+    
+    // Inner class for PortMapping
```

Comment:
This comment does not bring much value so I would remove it

Reply:
Removed